persistence/scaladsl/src/test/scala/com/lightbend/lagom/scaladsl/persistence/multinode/AbstractClusteredPersistentEntitySpec.scala

```diff
@@ -149,7 +149,7 @@ abstract class AbstractClusteredPersistentEntitySpec(config: AbstractClusteredPe
       enterBarrier("before-1")
 
       val ref1 = registry.refFor[TestEntity]("1").withAskTimeout(remaining)
-      val ref2 = registry.refFor[TestEntity]("2")
+      val ref2 = registry.refFor[TestEntity]("2").withAskTimeout(remaining)
```

Comment:
Hah! Good call.

I think remaining makes sense. I've reviewed the `within`s in this tests and I suspect we'd also need to increase the `Duration` provided for each test:

```scala
    "send commands to target entity" in within(21.seconds) {
```

I think `21.seconds` could be a bit too low in Travis. I use a more _naîve_ approach in https://github.com/lagom/lagom/pull/1369 and I think the `within(21.seconds)` will still be an issue.

Reply:
Heh, as soon as I saw your change I thought: yeah, probably better to increase the defaults than adding explicit timeouts everywhere!

Reply:
I should also add that `remaining` wasn't really my idea, I was just uncritically copying what was done for `ref1`. I actually am not sure that `remaining` makes sense, since it is a dynamically-calculated value that is being used in a static context here. Anyway, I think we can probably close this pull request, as I think you've nailed it in yours. Thanks!